### DISCUSSION TITLE
Updates?

### DISCUSSION THREAD

**Discussion by eSpadaMaddyS** [kyleskom/NBA-Machine-Learning-Sports-Betting#421] [Q&A]
Hi everyone. I watch for this project for a very long time,and i want to know about future updates. Do you plan to improve this project?Do you plan to support project after this game season? Or you think that this is the final version of your program?

**Comment by nealmick** [comment]
open source project, as long as we keep it going there isn't really a final version...

**Comment by kyleskom** [comment]
It's hard to say how much further this has with recent advancements in AI. I would like to explore those and see how those work into this project
